src/content/h1/tools/aether/readme.md

```diff
@@ -1,15 +1,219 @@
+
+<figure>
+  <a href="Omni_Lights.jpg">
+    <img src="Omni_Lights.jpg" alt="Omni lights added to light emitting faces"/>
+  </a>
+  <figcaption>
+    <p>Omni lights added to light emitting faces</p>
+  </figcaption>
+</figure>
+
+Now only your light emitting faces are visible. Create an omni light and set it up however you want. To place your lights, it is easiest to use 3D snaps, snapping to edge midpoints to get the light in the centre of a quad, but you can move them manually if you want. Once a light is in the right position make sure it is pushed out a little bit from the face so that the face itself is lit.
+
+You can move the pivot of the light before snapping them to faces so that you don't have to offset them individually. Make sure that for duplicate lights, you instance them rather than copy them so that should you want to make changes you can do so by only changing one.
+
+Now you should have your level filled with lights.
+
+Shadows are, again, largely down to use preferemce but it's easiest to turn on raytracing for all small lights with default settings, while using area shadows on the sky lights as they can produce nice soft shadows. Setting the area shadows length and width to ~200.0 works well for beavercreek.
+
+If you render your level now you'll probably notice that there's a lot of dark areas which isn't what you want, but those black areas will be handled by global illumination, using the Light Tracer. Open up the render dialog and go to the "Advanced Lighting tab. Select "Light Tracer" from the drop down list. There are a number of options that will affect your resulting lightmaps, but Rays Per Sample, Filter Size and Bounces are the important ones. A high rays per sample will create very accurate lighting, however this will increase your render time and since we are rendering to (relatively) small lightmaps, about 4 rays should be plenty. Filter size determines how much a ray should be scaled in order to merge together and produce a smooth result. If your light tracing looks blotchy, increase the filter size until it smooths out, or increase the ray count. The filter size was set at 8.0 for this tutorial. Bounces defines how many times a ray should reflect off of surfaces in the scene before stopping. This can seriously damage your render time so keep it small. Around 3 tends to be sufficient.
+
+These settings should be ample for lightmap usage as any more accuracy probably won't be noticable in the finished lightmap. If you're still getting black areas, some ambient light added in the Light Tracer settings can lift the really dark areas that might still occur. 
+
+**3ds MAX - importing lightmap**
```

Comment:
The "3ds MAX" prefix on each heading is a bit repetitive. I think it can be left out, and the `# Aether usage tutorial` heading could become `# Aether tutorial for 3ds Max`.